Added auto-semver-tag action (#1)
infobloxopen/auto-semver-tag: README.md

# Auto semver tag

> Action creates semver tag *(vX.Y.Z)* when pull request is merged. New version is calculated based on PR labels: "major", "minor". Patch version is incremented by default.

Reviewer: ```suggestion
Action creates semver tag *(vX.Y.Z)* when pull request is merged. New version is calculated based on presence of PR labels: "major", "minor". Otherwise patch version is incremented.
```
